data/src/main/java/com/linkedin/data/schema/validation/ValidationOptions.java

```diff
@@ -257,6 +257,36 @@ public boolean isAvroUnionMode()
     return _avroUnionMode;
   }
 
+  /**
+   * Set whether we should detect and fix base64 encoded fixed values.
+   *
+   * <p>Some clients cannot encode avro encoded strings, and send Base64 encoded fixed values instead. If this is
+   * set to true, then the validator attempts to detect this case and decode the base64 encoded string into fixed bytes
+   * to avoid breaking validation.</p>
+   *
+   * @param value True if base64 fixing mode for fixed values is enabled, false otherwise.
```

Comment:
`base64 fixing mode for fixed values` is reading awkward. Maybe "if flag to fix base64 encoding for fixed value is enabled"?